dl1_data_handler/image_mapper.py

```diff
@@ -0,0 +1,19 @@
+import numpy as np
+
+class ImageMapper():
+
+    def __init__(self, mapping_method, channels):#", settings=defaults, ..."):
+        self.mapping_method = mapping_method
+        self.channels = channels
+        self.mapping_tables = {}
+        self.image_shape = {
+            'LST_LSTCam': (110, 110, 1),
```

Comment:
Why these shapes?

Reply:
These shapes are the default ones and correspond to oversampling. E.g. if you rotate the pixel positions for the LSTCam so that they are aligned with the x- and y-axis, you will end up having 55 "positions" on one axis, which will be expand by the oversampling method by a factor of 2. With 55 positions (later it will declare as "ticks" in the code) I mean the number of elements (for one dimension) after applying np.unique() to the 2D pixel position array. Especially for the LSTCam the other axis has less than 110 ticks. To square the image, we fill extra rows or columns (depending on the axis) with artificial zeros. This will result in the typical stretch that the oversampling method is introducing to the images. With the new rebinning and interpolation methods we will get rid of this stretch. However, the image_mapper have an argument, where you can set an arbitrary image shape (only for the new methods), which will overwrite the default values above. Once this PR is merged, I will transfer the image mapping to DL1DH.